default.yml

```diff
@@ -22,6 +22,7 @@ Layout/LineLength:
 Lint/RaiseException:
   Enabled: true
 
+# TODO: Review this rule!
```

Comment:
Added some TODOs so we can make sure we don't forget these. But PRs will also be forthcoming.